community/kernel/src/main/java/org/neo4j/kernel/impl/api/security/AccessModeSnapshot.java

```diff
@@ -1,103 +0,0 @@
-/*
```

Comment:
`AccessModeSnapshot` is removed, because `AccessMode` is now immutable. Previously it was mutable because `AuthSubject` inherited from it. This is now handled using `SecurityContext`.